sql/donut.sql

```diff
@@ -0,0 +1,42 @@
+DROP TABLE IF EXISTS members;
+CREATE TABLE members (
+    member_id int(11) NOT NULL AUTO_INCREMENT, -- Primary Key
+    uid char(7) NOT NULL,  -- Unique
+    last_name varchar(255) NOT NULL, 
+    first_name varchar(255) NOT NULL,
+    middle_name varchar(255), 
+    email varchar(255) NOT NULL,
+    phone varchar(64) DEFAULT NULL,
+    gender enum('m', 'f') DEFAULT NULL, -- Question: are there PC options we need to add?
```

Comment:
This should be an integer. The enum should be stored as part of the codebase, not part of the database, so something like this:
```
class Gender:
  UNKNOWN = 0
  MALE = 1
  FEMALE = 2
  OTHER = 3
```
And you should separately have another column that optionally stores a string for when their gender is `Gender.OTHER`. Pronouns would be they, their, etc.